lib/transport/transport.ts

```diff
@@ -3,11 +3,21 @@
 
 import * as transportable from './transportable';
 import * as functionTransporter from './function-transporter';
+import * as v8TransportHelper from './v8-transport-helper';
```

Comment:
how about rename - 'v8-transport-helper' to 'builtin-object-transporter', to be a parallel concept of function-transporter.